Ginger/GingerCoreNETUnitTest/GingerRunnerTests/GingerRunnerTest.cs

```diff
@@ -53,8 +54,9 @@ public class GingerRunnerTest
         static SolutionRepository SR;
         static Solution solution;
         static ProjEnvironment environment;
+        static Context context;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'context' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4755016)